pkg/types/types.go

```diff
@@ -75,6 +75,51 @@ const (
 	EndpointKindRaw     EndpointKind = "raw"
 )
 
+type RuntimeName string
+
+func (r RuntimeName) String() string {
+	return string(r)
+}
+
+const (
+	RuntimeNameDocker     RuntimeName = "docker"
+	RuntimeNameContainerd RuntimeName = "containerd"
+	RuntimeNameCrio       RuntimeName = "cri-o"
+	RuntimeNamePodman     RuntimeName = "podman"
+	RuntimeNameUnknown    RuntimeName = "unknown"
+)
+
+func String2RuntimeName(name string) RuntimeName {
+	switch name {
+	case string(RuntimeNameDocker):
+		return RuntimeNameDocker
+	case string(RuntimeNameContainerd):
+		return RuntimeNameContainerd
+	case string(RuntimeNameCrio):
+		return RuntimeNameCrio
+	case string(RuntimeNamePodman):
+		return RuntimeNamePodman
+	}
+	return RuntimeNameUnknown
+}
```

Comment:
I prefer having specific types for the fields in the events so that people know what values to expect. Apart from the tests, this function is called in a few places, mainly to transform from flags to `RuntimeName` because in the core of the code all the functions use the `RuntimeName` type.